src/main/java/org/folio/rest/camunda/kafka/EventConsumer.java

```diff
@@ -1,36 +1,39 @@
-package org.folio.rest.camunda.jms;
+package org.folio.rest.camunda.kafka;
+
+import java.util.Map;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.util.Map;
 import org.camunda.bpm.engine.RuntimeService;
 import org.folio.spring.messaging.model.Event;
 import org.folio.spring.tenant.storage.ThreadLocalStorage;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.jms.annotation.JmsListener;
+import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.stereotype.Component;
 
 @Component
 public class EventConsumer {
 
   private static final Logger logger = LoggerFactory.getLogger(EventConsumer.class);
 
-  @Value("${event.queue.name}")
-  private String eventQueueName;
-
   @Autowired
   private RuntimeService runtimeService;
 
   @Autowired
   private ObjectMapper objectMapper;
 
-  @JmsListener(destination = "${event.queue.name}")
+  @KafkaListener(
+    id = "mod-camunda-events-listener",
+    containerFactory = "kafkaListenerContainerFactory",
```

Comment:
Everything else is an environment variable.
Should the `containerFactory` be one?
Should it specifically not be an environment variable configurable setting?
Is there only a single factory to select from?

Reply:
No. It is statically defined as in the change in spring-module-core, https://github.com/folio-org/spring-module-core/pull/68/files#diff-a1decc886717a788edfcdd290a58393320070338247431439e6e50000dccee82R27.